README.md

```diff
@@ -1,86 +1,67 @@
 ## Adding your node
-Submit a pull request to this repository adding the *multiaddress* of the IPFS node that your Ceramic node uses to the correct peerlist file. Peerlist files are organized by name of the Ceramic network. Currently, there are three peerlists:
-- **[`mainnet.json`](mainnet.json)**: A list of nodes that run the `mainnet` Ceramic network
-- **[`testnet-clay.json`](testnet-clay.json)**: A list of nodes that run the `testnet-clay` Ceramic network
-- **[`dev-unstable.json`](dev-unstable.json)**: A list of nodes that run the `dev-unstable` Ceramic network
-
-It is recommended to first run your node on the Clay testnet to ensure it is set up correctly before running mainnet.
-
-### Find your *multiaddress*
-When you start the Ceramic daemon you will get an output that looks something like this:
+Create an issue using the *Peerlist Request* template filling in each section with the neccessary information. 
 
-```sh
-$ ceramic daemon
-# ...
-Swarm listening on /ip4/127.0.0.1/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/192.168.188.22/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/10.70.198.64/tcp/4002/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-Swarm listening on /ip4/127.0.0.1/tcp/4003/ws/p2p/QmNZfs1Vevknvmykz3fWKTjhmEpckabhd2JyEGJuymZFsC
-```
-
-This is your IPFS multiaddress. It can also be viewed in the `config` file that the IPFS daemon creates which is located by default in a directory called `ipfs`. This directory gets created wherever IPFS runs. Of you are running IPFS out-of-process, you will see a similar log output on startup and you can simply get the multiaddress from there.
-
-It is recommended that you use a websocket multiaddress and configure SSL (e.g. `.../<port>/wss/p2p/<peerId>`). Before adding your multiaddress to the peerlist, be sure to replace the local IP addresses (e.g. `127.0.0.1`) with your external IP address or DNS name. Also ensure that the websocket port (`4003` in the example above) is open to internet connections. For more information see [multiaddress docs](https://github.com/multiformats/multiaddr).
+Once we receive your *Peerlist Request* issue we will review it and ask for more information if needed. When the *Peerlist Request* gets accepted we will add your IP address to our allow list. The *Peerlist Request* issue with then be closed and your node will be connected to the Ceramic network and the [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service). You can create a *Peerlist Request* issue for the following networks: 
 
-### Connectivity test
-When a pull request is submitted, it triggers a connectivity test to make sure your node is successfully connected to the network. If this fails, the 3Box Labs team will reach out to you to triage the issue. Make sure there are no firewalls blocking your instance and that your port is properly exposed.
+- **`mainnet`** 
```

Comment:
only mainnet will need anchor allowlist access.  Clay and dev-unstable have fully open CAS'